src/components/table/react/TableCell.tsx

```diff
@@ -0,0 +1,226 @@
+import React, { Fragment, useCallback } from 'react';
+
+import classNames from 'classnames';
+
+import { COMPONENT_PREFIX } from 'LumX/core/react/constants';
+import { IGenericProps, getRootClassName, validateComponent } from 'LumX/core/react/utils';
+import { handleBasicClasses } from 'LumX/core/utils';
+
+import { Icon } from 'LumX/components/icon/react/Icon';
+
+import { IconSizes } from 'LumX';
+import { mdiArrowDown, mdiArrowUp } from 'LumX/icons';
+
+import isFunction from 'lodash/isFunction';
+import noop from 'lodash/noop';
+
+/////////////////////////////
+
+/**
+ * The authorized values for the `sortOrder` prop.
+ */
+enum Orders {
+    asc = 'asc',
+    desc = 'desc',
+}
+type Order = Orders;
+
+/**
+ * The authorized values for the `scope` prop.
+ */
+enum Scopes {
+    col = 'col',
+    row = 'row',
+}
+type Scope = Scopes;
+
+/**
+ * The authorized values for the `scope` prop.
+ */
+enum Variants {
+    body = 'body',
+    head = 'head',
+}
+type Variant = Variants;
+
+/////////////////////////////
+
+/**
+ * Defines the props of the component.
+ */
+interface ITableCellProps extends IGenericProps {
+    /**
+     * The mdi name of the icon (thead only).
+     */
+    icon?: string;
+
+    /**
+     * Whether the column is sortable or not (thead only).
+     */
+    isSortable?: boolean;
+
+    /**
+     * The scope of the thead, possible values: col/row.
```

Comment:
Remove possible values, we will forgot to update the value in the js doc for sure